chopper/pubspec.yaml

```diff
@@ -1,28 +1,28 @@
 name: chopper
 description: Chopper is an http client generator using source_gen, inspired by Retrofit
-version: 3.0.6
```

Comment:
`version: 4.0.0-nullsafety.0`
we will probably release a non stable version first

Reply:
Ok added version: 4.0.0-nullsafety.0